### ISSUE TITLE
Incorrect tip color

### ISSUE THREAD

**Issue by dylemma** [qTip2/qTip2#753]
See the [fiddle](http://jsfiddle.net/rc4v7mcd/)

If you use a custom style on a tooltip, with no border, and whatever background color, the 'tip' seems to disregard the background color, choosing the nonexistent border color (which seems to be black). This is especially obvious with brighter background colors.

![image](https://cloud.githubusercontent.com/assets/1355958/8358947/440ff8da-1b31-11e5-8936-688f950f8fc2.png)

I've managed to work around the issue by explicitly setting the border-color to be the same as the background-color, but I do believe this is a bug.

I poked around in the code and found [these two lines](https://github.com/qTip2/qTip2/blob/master/src/tips/tips.js#L315-L316):

```javascript
// If border width isn't zero, use border color as fill if it's not invalid (1.0 style tips)
if(options.border && border < 1 && !INVALID.test(color[1])) { color[0] = color[1]; }
```

The comment seems to contradict the code, specifically the `border < 1` condition and the "If the border width isn't zero".

**Comment by zinkkrysty** [comment]
You are right, I also discovered that the code that is contradicting itself, but I am quite confused about what should be the right logic.

For me it doesn't make sense to set the bg color of the tip depending on a non-existing border. But I guess this is useful when falling back to taking the color from the other elements.

A test suite would make this issue a lot clearer.
